web-wallet/src/routes/(welcome)/setup/create/+page.svelte

```diff
@@ -41,12 +47,19 @@
 	/** @type {string[]} */
 	let enteredMnemonicPhrase = [];
 
+	const collectSettings = getKey("userId");
+	const userId = collectSettings($settingsStore);
```

Comment:
As you're retrieving properties as they are, you may want to just use destructuring here:

```js
const { userId } = $settingsStore;
```